public/manifest.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Knot.x Chrome Devtool Extension",
-  "version": "2.2.1-SNAPSHOT",
+  "version": "2.2.1",
```

Comment:
This should not be changed.

Reply:
should be. manifest cannot have letters in version name. Google return error after try to load extension with this name of version.